src/controller/CHIPDeviceController.cpp

```diff
@@ -404,6 +404,7 @@ CHIP_ERROR DeviceController::GetDevice(NodeId deviceId, const SerializedDevice &
         }
 
         device->Init(GetControllerDeviceInitParams(), mListenPort, mAdminId);
+        ReturnErrorOnFailure(device->LoadSecureSessionParametersIfNeeded());
```

Comment:
What specifically is this fixing?

Reply:
@pan-apple 
I found without this line, run following command may result in MsgCounter verify failure:
```
connect
zclread Cluster Attribute ...
zcl Cluster Command ...
```

After some research, I found the device get by GetDevice is not at a activate state. So I added this to set it to active before sending any commands.

Reply:
@pan-apple Could you please look over this part?